Update virtual-machine-network-throughput.md

steveesp opened the pull request:
a few tweaks to help with SEO - put "bandwidth" in the title and "VM" in the first paragraph

azurecla commented:
Hi __@jessebenson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jesseb). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

steveesp committed:
Update virtual-machine-network-throughput.md

a few tweaks to help with SEO - put "bandwidth" in the title and "VM" in the first paragraph
